osu.Game/Screens/Menu/ButtonSystem.cs

```diff
@@ -180,6 +181,20 @@ private void updateIdleState(bool isIdle)
                 State = ButtonSystemState.Initial;
         }
 
+        protected override bool OnKeyDown(KeyDownEvent e)
+        {
+            if (State == ButtonSystemState.Initial)
+            {
+                if (buttonsTopLevel.Any(b => e.PressedKeys.Contains(b.TriggerKey)))
```

Comment:
Is there a reason this isn't just `e.Key == `?

Reply:
no, that sounds better.